ehr_billing/src/org/labkey/ehr_billing/notification/BillingNotification.java

```diff
@@ -307,34 +313,22 @@ private void getProjectSummary(Container c, User u, final Calendar start, Calend
         final Map<FieldKey, ColumnInfo> cols = QueryService.get().getColumns(ti, fieldKeys);
         TableSelector ts = new TableSelector(ti, cols.values(), null, null);
         ts.setNamedParameters(params);
+        ChargeCategoryInfo categoryInfo = new ChargeCategoryInfo();
+        categoryInfo.setCategoryLabel(categoryName);
+        final double[] totalQuantity = {0.0}; // 1 element final array since 'final' is required in the below exec method
```

Comment:
This is fine - no need to change. FYI, org.apache.commons.lang3.mutable.MutableDouble is handy for scenarios too.

Reply:
I see, that's good to know! thank you